metrics/heapster.go

```diff
@@ -213,12 +212,12 @@ func getListersOrDie(kubernetesUrl *url.URL) (*cache.StoreToPodLister, *cache.St
 	return podLister, nodeLister
 }
 
-func createKubeClientOrDie(kubernetesUrl *url.URL) *kube_client.Client {
+func createKubeClientOrDie(kubernetesUrl *url.URL) *kube_client.Clientset {
```

Comment:
This looks dirty.  doesn't have to be fixed here, but why do you have a side-channel to load the server location instead of a kubeconfig file?